src/components/i18n/__tests__/__snapshots__/i18n.test.js.snap

```diff
@@ -1663,6 +1695,14 @@ Array [
     "file": "./src/components/scans/scans.js",
     "key": "view.empty-state",
   },
+  Object {
+    "file": "./src/components/scans/scansContext.js",
+    "key": "form-dialog.confirmation",
+  },
+  Object {
+    "file": "./src/components/scans/scansContext.js",
+    "key": "form-dialog.confirmation",
+  },
```

Comment:
This appears to be duplicated. Is that correct?

Reply:
Indeed, if you ref the file `src/components/scans/scansContext.js` and do a string search on the file you'll see `form-dialog.confirmation` pop up twice. 

The i18n snapshots @nicolearagao are a reference point for knowing where all keys are. It doesn't necessarily tell you if something is busted but like all JS snapshots in this repo its the first line of alert against "something changed, why did it change, did I mean to change that..." =) At some point I need to update the`CONTRIBUTING.md` with process how and whys.